cli/script/command-executor.ts

```diff
@@ -1071,6 +1071,7 @@ function promote(command: cli.IPromoteCommand): Promise<void> {
     var packageInfo: PackageInfo = {
         appVersion: command.appStoreVersion,
         description: command.description,
+        label: command.label,
```

Comment:
Can you update the `log("Successfully ...` line below to mention the label that was promoted (only if it was specified by the user)?